src/features/common/Layout/ProjectsLayout/index.tsx

```diff
@@ -29,7 +34,11 @@ const ProjectsLayout: FC<ProjectsLayoutProps> = ({
           <main className={styles.mainContent}>
             <section className={styles.contentContainer}>{children}</section>
             <section className={styles.mapContainer}>
-              <ProjectsMap />
+              <ProjectsMap
+                isMobile={false}
+                selectedMode={selectedMode}
+                setSelectedMode={setSelectedMode}
```

Comment:
In the desktop layout, does `selectedMode` matter? If not, perhaps we should not pass it in?